packages/webservice/app/src/main/resources/application.yaml

```diff
@@ -49,6 +49,10 @@ logging:
   path: /tmp
 
 ---
+elide:
+  async:
+    cleanupEnabled: false
```

Comment:
do we need a profile here?

Reply:
no, anything below `---` gets tagged with test profile.